src/query/list_file.c

```diff
@@ -414,6 +415,32 @@ qfile_clone_list_id (const QFILE_LIST_ID * list_id_p, bool is_include_sort_list)
 void
 qfile_clear_list_id (QFILE_LIST_ID * list_id_p)
 {
+#if !defined(NDEBUG)
+  QMGR_QUERY_ENTRY *query_p;
+  TDE_ALGORITHM tde_algo = TDE_ALGORITHM_NONE;
+  THREAD_ENTRY *thread_p = thread_get_thread_entry_info ();
+  int tran_index;
+
+  tran_index = LOG_FIND_THREAD_TRAN_INDEX (thread_p);
+
+  query_p = qmgr_get_query_entry (thread_p, list_id_p->query_id, tran_index);
+  if (query_p != NULL && query_p->includes_tde_class)
```

Comment:
`qmgr_find_query_entry()` is static function, so it can't be used here. 
As you said, `qmgr_get_query_entry()` could create a query entry and load from a session entry. Could it matter?

Reply:
Your review has got me to know the existence of the session entry, `SESSION_QUERY_ENTRY`. It seems to be made based on a query entry in `qentry_to_sentry()`, which may be related to a holdable entry, and it can be used as a source for making a query entry. Do you know whether this entry has to include tde information or not?

Reply:
For note: He said I possibly should free the query entry created from the session entry.
I'll consider it.

Reply:
> Your review has got me to know the existence of the session entry, SESSION_QUERY_ENTRY. It seems to be made based on a query entry in qentry_to_sentry(), which may be related to a holdable entry, and it can be used as a source for making a query entry. Do you know whether this entry has to include tde information or not?

I've confirmed that the tde information is already preserved.

Reply:
I'll remove this debug code it doesn't seem proper location to test it